Ecommerce/index.html

```diff
@@ -1,13 +1,83 @@
-<!DOCTYPE html>
```

Comment:
Remove the console.log statement as it is unnecessary in an HTML file and can clutter the console.